tests/spec/core/xref-spec.js

```diff
@@ -282,6 +282,49 @@ describe("Core — xref", () => {
     expect(five.title).toBe("Error: No matching dfn found.");
   });
 
+  it("uses data-cite fallbacks", async () => {
+    const body = `
+      <section data-cite="dom html" id="test">
+        <section><dfn>local</dfn></section>
+        <p><a id="link1">event handler</a> try either [dom] or [html]</p>
+        <section data-cite="dom">
+          <p data-cite="svg">
+            <a id="link2">event handler</a>
+            - not in [svg] -> fallback to [dom] -> fallback to [dom], [html]
```

Comment:
would it be worth adding `<a>local</a>` here?

Reply:
Sure